backend/manager/modules/utils/src/main/java/org/ovirt/engine/core/utils/ovf/OvfOvaVmWriter.java

```diff
@@ -60,6 +60,7 @@ protected void writeGeneralData() {
         if (vm.getCpuPinning() != null) {
             _writer.writeElement(CPU_PINNING, vm.getCpuPinning());
         }
+        writeNumaNodeList();
```

Comment:
wouldn't it make more sense to have it in writeHardware like memory, cpu, etc?

Reply:
I tried to align with the current implementation of the `OvfVmWriter`. It makes sense that NUMA are part of the hardware but in general we also put it under the host (we can also say it semi resource allocation related).

Reply:
it is but like the CPU - you can define pinning but you can also define the resources without pinning and then it's unrelated to the host. it doesn't matter that much because I couldn't find NUMA in the OVF specification but still ..

Reply:
I did it only to the OVA because i didn't want to touch the existing flows.